core/store/benches/trie_bench.rs

```diff
@@ -44,7 +42,7 @@ fn trie_update(bench: &mut Bencher) {
 
     bench.iter(|| {
         let mut this_changes = changes.clone();
-        let (_, _) = trie.update(&root, this_changes.drain(..));
+        let _ = trie.update(&root, this_changes.drain(..));
```

Comment:
Why is `let _` necessary?

Reply:
otherwise there is warning about unused value.

Reply:
I see